frontend/cypress/e2e/AdminE2E/userManagement.cy.js

```diff
@@ -0,0 +1,262 @@
+import LoginPage from "../../pages/LoginPage";
+
+let loginPage = null;
+let homePage = null;
+let adminPage = null;
+let userMgmnt = null;
+let usersData;
+
+before(() => {
+  // Initialize LoginPage object and navigate to Admin Page
+  loginPage = new LoginPage();
+  loginPage.visit();
+
+  homePage = loginPage.goToHomePage();
+  adminPage = homePage.goToAdminPage();
+});
+
+describe("User Management", function () {
+  beforeEach(() => {
+    cy.fixture("UserManagement").then((users) => {
+      usersData = users;
+    });
+  });
+
+  it("Navigate to User Management Page", function () {
+    userMgmnt = adminPage.goToUserManagementPage();
+    userMgmnt.verifyPageTitle();
+  });
+
+  describe("Add User and Exit", function () {
+    it("Add User", function () {
+      userMgmnt.clickAddButton();
+      userMgmnt.validatePageTitle();
+    });
+
+    it("Enter USer details", function () {
+      userMgmnt.typeLoginName(usersData[0].username);
+      userMgmnt.passwordExpiryDate(usersData[0].passwordExpiryDate);
+      userMgmnt.typeLoginPassword(usersData[0].password);
+      userMgmnt.repeatPassword(usersData[0].password);
+      userMgmnt.enterFirstName(usersData[0].fName);
+      userMgmnt.enterLastName(usersData[0].lName);
+      userMgmnt.enterUserTimeout(usersData[0].userTimeout);
+    });
+
+    it("Add and Remove Lab Unit Roles", function () {
+      userMgmnt.addNewPermission();
+      userMgmnt.allPermissions();
+      userMgmnt.removePermission();
+    });
+
+    it("Apply Roles and Permissions", function () {
+      //userMgmnt.copyPermisionsFromUser();
+      //userMgmnt.applyChanges();
```

Comment:
thanks @Bahati308 for the work on this .. 
could we remove the comment out code lines and in other places